{
  try {
    ff.function(functionName,ff.literal(13));
  }
 catch (  RuntimeException e) {
    assertEquals("Unable to find function " + functionName,e.getMessage());
    return;
  }
  fail("Exception not thrown");
}
